### ISSUE TITLE
[Bug] Failed to Import BCH Wallet from a seed created on bitcoin.com 

### ISSUE THREAD

**Issue by mvozzo** [cypherstack/stack_wallet#189]
Hi Stack Wallet team, 
I tried importing a BCH Wallet that I created on Bitcoin,com with a 12 word seed. 
I was able to import the wallet fine in the Bitcoin.com Mobile Wallet App. 
But when importing the 12 word seed phrase into Stack Wallet, the seed is accepted but the balance = 0
FYI, the Derivation Path for the BCH Wallet = 44'/0'/0'  
I think there may be a bug with the import, or is the default derivation path being set to something different. If this is the case, please allow in the Advanced Wallet settings the ability for the user to set a Custom Derivation Path.  Many Thanks.

**Comment by samedamci** [comment]
I cannot import any seed. Tried Bitcoin and Litecoin. Seed is correct, even wallet highlight words to green color, but after all it says the seed phrase is incorrect. Only Wownero seed worked, but after syncing it doesn't show any transaction and balance is 0.

**Comment by sneurlax** [comment]
@wzykubek, if you still have issues, would you please open a new issue?  I've implemented a fix for this issue's topic and will be closing it.  It sounds like you were having connectivity issues to me, but please do create a new issue to track your problems and we will address them.

@mvozzo, Stack Wallet uses the SLIP44 derivation path, m/44/145, for P2PKH addresses.  Bitcoin.com uses BIP44, m/44/0.  I added code during wallet restoration to check both SLIP44 and BIP44 addresses during restoration.  Stack Wallet should detect and use Bitcoin.com-wallet-created outputs after #330 is tested and merged

**Comment by rehrar** [comment]
Fixed
